ui.tests/test-module/specs/fragmentcontainer.spec.js

```diff
@@ -0,0 +1,124 @@
+
+        //since data model is already selected it should be disabled
+        cy.get(".cmp-adaptiveform-container__selectformmodel").should("not.have.attr", "disabled");
+        cy.get(".cmp-adaptiveform-container__fdmselector").click();
+
+        cy.get("coral-selectlist-item[value='/content/dam/formsanddocuments-fdm/forms-ootb-usc-workflow-fdm']").contains('Workflow Unified Storage Form Data Model').should('be.visible').click();
+        cy.get("#formModelChange").should("be.visible");
+        cy.get("#formModelDialogAcceptButton").click();
+        // Not undo the changes
+        cy.get(".cmp-adaptiveform-container__selectformmodel").click();
+        cy.get("coral-selectlist-item[value='none']")
+            .contains('None')
+            .click();
+        cy.get(".cq-dialog-submit").click();
+    };
+
+    context("Open Forms Editor", function () {
+        const pagePath = "/content/forms/af/core-components-it/samples/fragment/test-fragment",
+            fragmentContainerEditPath = pagePath + afConstants.FORM_EDITOR_FORM_CONTAINER_SUFFIX,
+            fragmentContainerEditPathSelector = "[data-path='" + fragmentContainerEditPath + "']",
+            fragmentContainerDropPath = pagePath + afConstants.RESPONSIVE_GRID_SUFFIX + "/" + afConstants.components.forms.resourceType.fragmentcontainer.split("/").pop();
+        beforeEach(function () {
+            // this is done since cypress session results in 403 sometimes
+            cy.openAuthoring(pagePath);
+        });
+
+        it('open edit dialog of adaptive form fragment container component', function () {
+            checkEditDialog(fragmentContainerEditPathSelector);
+            cy.get('.cq-dialog-cancel').click();
+        });
+
+        it('open and select data model in fragment container edit dialog box', function () {
+            verifyOpenDataModel(fragmentContainerEditPathSelector);
+        });
+
+        it('change data model in fragment container edit dialog box', {retries: 3},function () {
+            verifyChangeDataModel(fragmentContainerEditPathSelector);
+        });
+
+        it ('check tabs and title in edit dialog', {retries: 3}, function() {
+            checkTitleInEditDialog(fragmentContainerEditPathSelector);
+            // stringent check on fragment container edit dialog tabs so that introducing new tab
+            // in form container edit dialog should force author to check if that is required for fragment
+            cy.get(".cmp-adaptiveform-container__editdialog coral-tab").should("have.length", 2);
+            cy.get(".cmp-adaptiveform-container__editdialog coral-tab-label:contains('Basic')")
+                .should("exist");
+            cy.get(".cmp-adaptiveform-container__editdialog coral-tab-label:contains('Data Model')")
+                .should("exist");
+            cy.get(".cmp-adaptiveform-container__editdialog coral-tab-label:contains('Submission')")
```

Comment:
Can you try avoiding use of coral selector in the test, in fact rely on only our selectors ?